fabric-dimensions-v1/src/main/java/net/fabricmc/fabric/api/dimension/v1/FabricDimensionType.java

```diff
@@ -16,10 +16,12 @@
 
 package net.fabricmc.fabric.api.dimension.v1;
 
+import java.util.function.BiFunction;
```

Comment:
rogue import?

Reply:
Not a new import.  Used in several places.  Looks like my IDE moved it to top vs where it was before. (At bottom.) Will move it back.